pkg/util/k8sutil/k8sutil.go

```diff
@@ -117,9 +121,17 @@ func PodWithNodeSelector(p *v1.Pod, ns map[string]string) *v1.Pod {
 	return p
 }
 
-func CreateClientService(kubecli kubernetes.Interface, clusterName, ns string, owner metav1.OwnerReference) error {
+func CreateClientService(kubecli kubernetes.Interface, clusterName, ns string, isSecureClient bool, owner metav1.OwnerReference) error {
```

Comment:
No need to change this func?

Reply:
I know, but I chose to change it to make things consistent.